solr/solrj/src/java/org/apache/solr/client/solrj/request/beans/SplitCorePayload.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.solr.client.solrj.request.beans;
+
+import org.apache.solr.common.annotation.JsonProperty;
+import org.apache.solr.common.util.ReflectMapWriter;
+
+import java.util.List;
+
+public class SplitCorePayload implements ReflectMapWriter {
+    // Should be converted as an array
```

Comment:
is this a bug that we need to fix elsewhere, not in the bounds of this PR, or is this more of a wish?  Is there a SOLR- that could be referenced?

Reply:
Oops - no, this isn't a bug.  This was just a "note to self" so I could remember some niggling detail while I was working on this one.  I forgot to remove it is all.  Will remove.